ecolex/management/commands/legislation.py

```diff
@@ -25,111 +25,85 @@ def __init__(self, config):
-
-                if not legislation:
-                    logger.error('Failed to find legislation %s' % (obj.doc_id))
-                    obj.status = DocumentText.FULL_INDEX_FAIL
-                    obj.save()
-                    continue
-
-                legislation['legText'] = text
-                result = self.solr.add(legislation)
-                if result:
-                    logger.info('Success download & indexed: %s' % (obj.doc_id,))
-                    obj.status = DocumentText.FULL_INDEXED
-                    obj.doc_size = doc_size
-                    obj.text = text
-                    try:
-                        obj.save()
-                    except OperationalError as e:
-                        logger.error("DB insert error %s %s" % (obj.doc_id, e))
-                        obj.status = DocumentText.FULL_INDEX_FAIL
-                        obj.text = None
-                        obj.save()
-
-                else:
-                    logger.error('Failed doc extract %s %s' % (obj.url,
-                                                               legislation['id']))
-        logger.info('[Legislation] Update full text finished.')
-
-    def reindex_failed(self):
-        logger.info('[legislation] Reindex started.')
-        objs = DocumentText.objects.filter(
-            status=DocumentText.INDEX_FAIL, doc_type=LEGISLATION)
-        if objs.count() > 0:
-            for obj in objs:
-                legislation = json.loads(obj.parsed_data)
-                legislation['updatedDate'] = (datetime.now()
-                                              .strftime('%Y-%m-%dT%H:%M:%SZ'))
-                # Check if already present in Solr
-                if 'id' not in legislation:
-                    try:
-                        old_legislation = self.solr.search(LEGISLATION, obj.doc_id)
-                        if old_legislation:
-                            legislation['id'] = old_legislation['id']
-                            legislation = cleanup_copyfields(legislation)
-                    except SolrError as e:
-                        logger.error('Error reading legislation %s' % (obj.doc_id,))
-                        if settings.DEBUG:
-                            logging.getLogger('solr').exception(e)
-                        continue
-                result = self.solr.add(legislation)
-                if result:
-                    obj.status = DocumentText.INDEXED
-                    obj.parsed_data = ''
-                    obj.save()
-                    logger.info('Success indexing: %s' % (obj.doc_id,))
-                else:
-                    logger.error('Failed to index: %s' % (obj.doc_id,))
-        logger.info('[legislation] Reindex finished.')
+    def update_full_text(self, obj):
+        if not obj:
+            return
+        # Check if already parsed
+        text = None
+        if obj.doc_size and obj.text:
+            logger.info('Checking content length of %s (%s)' %
```

Comment:
Please change string formatting to [f-strings](https://docs.python.org/3/tutorial/inputoutput.html#tut-f-strings) throughout the project (or at least in the files you modify).